src/extension/features/general/hide-help/index.js

```diff
@@ -1,51 +1,62 @@
 import { Feature } from 'toolkit/extension/features/feature';
 import { getToolkitStorageKey, setToolkitStorageKey } from 'toolkit/extension/utils/toolkit';
+import { getEmberView, controllerLookup } from 'toolkit/extension/utils/toolkit';
 
 export class HideHelp extends Feature {
   injectCSS() { return require('./index.css'); }
 
```

Comment:
Is this still needed? Looks like index.css has been deleted. The Tavis build errors also indicate the file cannot be found.

Reply:
woops -- nope.